src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+## Lazy-load third-party resources
+
+Embedded third-party resources can be a big contributor to slow page speed when constructed poorly. If they aren’t critical or are below the fold (that is, if users have to scroll to view them), lazy-loading is a good way to improve page speed and paint metrics. This way users will get the main page content faster and have a better experience.
+
+<figure class='w-figure w-figure--inline-left'>
+<img src="lazy-load.png" alt="A diagram of a webpage shown on a mobile device with scrollable content extending beyond the screen. The content that's below-the-fold is desaturated because it's not loaded yet.">
+</figure>
+
+One way to do that is to lazy-load third-party content after the main page content loads. Ads are a good candidate for this approach.
+
+Ads are an important source of income for many sites, but users come for the content. By lazy-loading ads and delivering the main content faster, you can increase the overall viewability percentage of an ad. For example, MediaVine switched to [lazy-loading ads](https://www.mediavine.com/lazy-loading-ads-mediavine-ads-load-200-faster/) and saw a 200% improvement in page load speed. DoubleClick have guidance on how to lazy-load ads in their [official documentation](https://support.google.com/dfp_premium/answer/4578089#lazyloading). 
+
+An alternative approach is to load third-party content only when users scroll down to that section of the page.
+
+[Intersection Observer](https://developers.google.com/web/updates/2016/04/intersectionobserver) is a browser API that efficiently detects when an element enters or exits the browser's viewport and it can be used to implement this technique.
+
+[lazysizes](https://web.dev/use-lazysizes-to-lazyload-images/) is a popular JavaScript library for lazy-loading images and [`iframes`](http://afarkas.github.io/lazysizes/#examples). It supports YouTube embeds and [widgets](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/unveilhooks). It also has [optional support](https://github.com/aFarkas/lazysizes/blob/097a9878817dd17be3366633e555f3929a7eaaf1/src/lazysizes-intersection.js) for IntersectionObserver. 
+
+{% Aside 'caution' %}
+
+Be careful when lazy-loading resources with JavaScript. If JavaScript fails to load, perhaps due to flaky network conditions, your resources won’t load at all. Keep an eye out for [native lazy-loading support for images and iframes](https://addyosmani.com/blog/lazy-loading/) that’s coming soon to Chrome.
+
+{% endAside %}
+
+## Optimize how you serve third-party scripts
+
+
+### Use third-party CDN hosting
```

Comment:
Sorry to keep coming back to this issue, but I want to make sure the advice we're providing is clear. Based on the sections that came before, this heading makes it seem like we're proposing CDNs as a viable option. But we're not really, at least as I read the post. If we think people should use self-hosting or CDNs with a service worker, I kinda think we should just come out and say that. @katejeffreys and @mihajlija, what do y'all think?